source/NeutronaWand/Header.h

```diff
@@ -305,6 +305,13 @@ bool b_sound_firing_cross_the_streams = false;
 bool b_sound_idle = false;
 bool b_beeping = false;
 
+/*
```

Comment:
Nice, I should've looked to see if Arduino supported prototypes or "forward" declarations. Didn't realize it did.